schema/plotData.json

```diff
@@ -0,0 +1,79 @@
+{
+  "type": "object",
+  "description": "Daily plot data and metadata for a specific region ID and variable ID (found in filename by convention).",
+  "properties": {
+    "metadata": {
+      "properties": {
+        "minYear": {
+          "type": "integer",
+          "description": "The year in which the minimum for this region and variable occurred."
+        },
+        "maxYear": {
+          "type": "integer",
+          "description": "The year in which the maximum for this region and variable occurred."
+        }
+      },
+      "required": ["minYear", "maxYear"],
+      "additionalProperties": false
+    },
+    "data": {
+      "type": "object",
+      "description": "A mapping of column headers to arrays of column values. It's expected that each array will be the same length.",
+      "properties": {
+        "dayOfWaterYear": {
+          "type": "array",
+          "items": { "type": "integer", "minimum": 1, "maximum": 366 },
+          "description": "The day of water year for this row, starting with 1."
+        },
+        "date": {
+          "type": "array",
+          "items": { "type": "string", "format": "date" },
+          "description": "The calendar date (YYYY-MM-DD) for this row."
+        },
+        "yearToDate": {
+          "type": "array",
+          "items": { "type": ["number", "null"] },
+          "description": "Observed values for the current water year to date."
+        },
+        "min": {
+          "type": "array",
+          "items": { "type": ["number", "null"] },
+          "description": "Observed values for the minimum water year (as specified in `metadata.minYear`)."
+        },
+        "max": {
+          "type": "array",
+          "items": { "type": ["number", "null"] },
+          "description": "Observed values for the maximum water year (as specified in `metadata.maxYear`)."
+        },
+        "median": {
+          "type": "array",
+          "items": { "type": ["number", "null"] },
+          "description": "Climatological median for each day of the water year."
```

Comment:
Climatological may be confusing, maybe say "median over the full historic dataset for each day of the water year"

Reply:
What do you think of:

> Historical median for each day of the water year

?

Reply:
f3ddd75